sources/packages/web/src/views/aest/student/ViewScholasticStanding.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <header-navigator
```

Comment:
The `full-page-container` has a specific slot for the header (#header), please move it inside.